File: second.py
# Fibonacci series up to n terms
n_terms = int(input("Enter the number of terms: "))

# Initial two terms
n1, n2 = 0, 1
count = 0

if n_terms <= 0:
    print("Please enter a positive integer.")
elif n_terms == 1:
    print(f"Fibonacci sequence: {n1}")
else:
    print("Fibonacci sequence:")
    while count < n_terms:
        print(n1, end=" ")
        nth = n1 + n2
        # Update values
        n1 = n2
        n2 = nth
        count += 1

File: second.c
#include <stdio.h>

int main() {
    int i, n;
    // Initialize first and second terms
    int t1 = 0, t2 = 1;
    // Initialize the next term (3rd term)
    int nextTerm = t1 + t2;

    printf("Enter the number of terms: ");
    scanf("%d", &n);

    // Handle 0 or negative terms
    if (n <= 0) {
        printf("Please enter a positive integer.\n");
        return 0;
    }

    printf("Fibonacci Series: ");

    for (i = 1; i <= n; ++i) {
        printf("%d ", t1);
        nextTerm = t1 + t2;
        t1 = t2;
        t2 = nextTerm;
    }

    return 0;
}

